<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LecteurVue</class>
 <widget class="QMainWindow" name="LecteurVue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mon Application</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="titre">
      <property name="text">
       <string>Titre</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="image">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Image</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="categorie">
        <property name="text">
         <string>Catégorie</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="rang">
        <property name="text">
         <string>Rang</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="precedent">
        <property name="text">
         <string>Précédent</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="lancer_diaporama">
        <property name="text">
         <string>Lancer Diaporama</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="suivant">
        <property name="text">
         <string>Suivant</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>36</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Fichier">
    <property name="title">
     <string>&amp;Fichier</string>
    </property>
    <addaction name="actionSortir"/>
   </widget>
   <widget class="QMenu" name="menuParam_tre">
    <property name="title">
     <string>&amp;Paramètre</string>
    </property>
    <widget class="QMenu" name="menuVItesse_de_d_filement">
     <property name="title">
      <string>VItesse de défilement</string>
     </property>
     <addaction name="actionToutes_les_2_secondes"/>
    </widget>
    <addaction name="actionChanger_diaporama"/>
    <addaction name="actionEnlever_diaporama"/>
    <addaction name="menuVItesse_de_d_filement"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="action_propos_de"/>
   </widget>
   <addaction name="menu_Fichier"/>
   <addaction name="menuParam_tre"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_quitter">
   <property name="text">
    <string>Sortir</string>
   </property>
  </action>
  <action name="action_changer_diaporama">
   <property name="text">
    <string>Changer diaporama</string>
   </property>
  </action>
  <action name="action_enlever_diaporama">
   <property name="text">
    <string>Enlever diaporama</string>
   </property>
  </action>
  <action name="action_vitesse_defilement">
   <property name="text">
    <string>Vitesse de défilement</string>
   </property>
  </action>
  <action name="actionSortir">
   <property name="text">
    <string>Sortir</string>
   </property>
  </action>
  <action name="actionChanger_diaporama">
   <property name="text">
    <string>Charger diaporama</string>
   </property>
  </action>
  <action name="actionEnlever_diaporama">
   <property name="text">
    <string>Enlever diaporama</string>
   </property>
  </action>
  <action name="actionToutes_les_2_secondes">
   <property name="text">
    <string>Toutes les 2 secondes</string>
   </property>
  </action>
  <action name="action_propos_de">
   <property name="text">
    <string>À propos de...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
